.circleci/config.yml

```diff
@@ -561,7 +587,7 @@ workflows:
       # Always run basic test/lint/compilation (open PRs, master merge).
       # Note: by default tags are not picked up.
       - api-local-test
-      - api-build-test
+      - api-unit-test
       - ui-build-test
```

Comment:
nit: is there a better name for `ui-build-test`?

Reply:
🤔 `ui-bvt`? At IBM, `BVT` is universally accepted short name for "Build Verification Tests". But word of caution, I don't find any team in Broad use it.

Reply:
I think build is implied. I don't know of any non-build tests we'd be doing here (except linting).

What do you call this suite? Maybe `angular-test`?

Reply:
It checks for both compilation and runs the unit tests. Full compilation is not required to run the unit tests, so I don't agree that it's implicit, though I'd be fine with just ui-test. angular-test would be inaccurate.

Reply:
I see the other was renamed to unit. That would be fine here as well, `ui-unit-test`